web/src/components/namespace/project-cards/project-card-dropdown.tsx

```diff
@@ -29,6 +29,10 @@ export const ProjectCardDropdown: FC<Props> = (props) => {
 
   const [localStarred, setLocalStarred] = useState(isStarred);
 
+  useEffect(() => {
+    setLocalStarred(isStarred);
+  }, [project]);
+
```

Comment:
Oh, wait I know how to fix this one.... Wherever you use `<ProjectCardDropdown />` add the prop `key={isStarred}` and it will rerender, and then you will no longer need this `useEffect`

Reply:
Fixed